### DISCUSSION TITLE
User interface Safari iOS?

### DISCUSSION THREAD

**Discussion by janreinhardt** [nicolargo/glances#2986] [Q&A]
Please, I cleared the browser cache on my iPad Air and afterwards Glances does not look like it looked before.
Where has the usual user interface gone (dark black and green), why is the font different now? Is this Safari / iPad behaviour? How do I get Glances to look like it used to do?
(phone and Mac Safari web layout unchanged, only on iPad Air things look all funny-crazy)

[
![IMG_1931](https://github.com/user-attachments/assets/8d9daccf-caaf-47e1-91f7-9920263c5d57)
](url)

**Comment by rkison** [comment]
I'm also experiencing the same and would like to have the web UI display in a monospaced font. I've reverted to 4.1.2.1 for now.

macOS 15.1 with Safari 18.1

**Comment by janreinhardt** [comment]
Rolled back to 4.1.2.1 as well: does not solve it. On iPad Air (latest iOS) colours went white, but user interface and font are still screwed. 
But also: everything is fine on iPhone (latest iOS) and on Mac (latest macOS). 


![IMG_1932](https://github.com/user-attachments/assets/680a733a-b4ee-46be-a781-5281e2071810)

**Comment by janreinhardt** [comment]
Ive also tried with Firefox: same mess. So this is not a Safari browser thing, but more a general iPad Air (Screen resolution?) iOS 18.0.1 thing?

![IMG_1933](https://github.com/user-attachments/assets/b4fba8b9-fde9-4b77-aafd-64bee3eb4172)

**Comment by Drinkmelk** [comment]
Same issue here on all browsers. Rolled back to 4.1.1 for now.

**Comment by bluecasita** [comment]
I'm having the same issue.  Was there an update to the UI in 4.2.0?

**Comment by janreinhardt** [comment]
@Drinkmelk and @Matt-UKTX youre on iPad Air as well I assume?

**Reply by Drinkmelk** [reply]
No, MacOS and Windows 11. Tried both Firefox and Chrome but no luck.

**Reply by bluecasita** [reply]
Same for me.  MacOS and Windows 11, both Safari and Chrome.  I downgraded to 4.1.1, cleared the cache, and the issue went away.  Waiting to hear if this is some kind of UI update or a bug.

**Comment by janreinhardt** [comment]
@nicolargo so is this a real Glances bug then? Should this remain a Github „Discussion“ or move over to „Issues“ ?

**Comment by janreinhardt** [comment]
fuck me, it has switched back to normal! 
As of right now, the Glances UI on my iPad Air looks perfectly fine again. And I didnt touch or change anything.

**Comment by janreinhardt** [comment]
just updated to Glances 4.2.1: works like a charm, font and black background look ace
Thank you @nicolargo

**Reply by bluecasita** [reply]
I upgraded too.  Text is still wrapping in the "Command" section which creates weird line spacing.  How do we get it to look like it did on 4.1.1?

**Reply by the-night-wing** [reply]
same for me, weird wrapping and containers section is somewhere at the bottom
